### PR TITLE
Fixed bug, when the `Chocies.validator` was producing the wrong output.

### LINKED ISSUE #35: Invalid exception formatting within Chocies.validator
I'm' sorry to spam you with issues like that :^) I noticed a small bug in Choices.validator, you don't pass the value to exception:

```
python3.4/site-packages/djchoices/choices.py in __call__(self, value)
    118     def __call__(self, value):
    119         if value not in self.values:
--> 120             raise ValidationError('Select a valid choice. %(value)s is not '
    121                                   'one of the available choices.')
    122 

ValidationError: ['Select a valid choice. %(value)s is not one of the available choices.']
```

### PR DESCRIPTION
**Pull request by sobolevn**
Just a small fix with passing the value into formating.
Closes #35

### CODE REVIEW

**Comment by sergei-maertens** [comment]
Thanks, I forgot to look at this! Could you maybe add modify a test for this change?

**Comment by sergei-maertens** [comment]
@sobolevn can you make the PR against `develop` instead of `master` as well?

File: djchoices/tests/test_choices.py [hunk 1]
```diff
@@ -125,6 +125,16 @@ def test_numeric_class_validator(self):
         self.assertRaises(ValidationError, NumericTestClass.validator, 6)
         self.assertRaises(ValidationError, NumericTestClass.validator, 7)
 
+    def test_validation_error_message(self):
+        from django.core.exceptions import ValidationError
+
+        message = ("Select a valid choice. 4 is not "
+                   "one of the available choices.")
+
+        with self.assertRaises(ValidationError) as context:
```
Comment (sergei-maertens) [inline djchoices/tests/test_choices.py]: you can also use `with self.assertRaisesMessage(ValidationError, 'Select a valid choice. 4 is not one of the available choices'):`

Comment (sobolevn) [inline djchoices/tests/test_choices.py]: nope, that's not working.

**Pull request closed without merging**